master/internal/task/allocation.go

```diff
@@ -111,7 +111,7 @@ const (
 const (
 	killCooldown       = 30 * time.Second
 	okExitMessage      = "command exited successfully"
-	missingExitMessage = "command exit reason missing"
+	missingExitMessage = ""
```

Comment:
I feel like this may as well just be a literal in the spot where it's used now. Either way.